docs/sources/datasources/influxdb/query-editor/index.md

```diff
@@ -179,12 +188,12 @@ Examples:
 ## Flux query editor
 
 Grafana supports Flux when running InfluxDB v1.8 and higher.
-If your data source is [configured for Flux]({{< relref "./#configure-the-data-source" >}}), you can use
-the [Flux query and scripting language](https://www.influxdata.com/products/flux/) in the query editor, which serves as
+If your data source is [configured for Flux](/docs/grafana/latest/datasources/influxdb/configure-influxdb-data-source/), you can use
```

Comment:
This should be a relative path or a `ref` URI so that Grafana Cloud readers  in https://grafana.com/docs/grafana-cloud/connect-externally-hosted/data-sources/influxdb/ aren't directed back to Grafana docs

Reply:
fixed - thanks @jdbaldry